packages/theme/src/core/button/types.ts

```diff
@@ -2,7 +2,7 @@ import { FontVariants } from '../font/types';
 
 export type ButtonSizes = 'S' | 'M' | 'L';
 
-type Colors = {
+export type Colors = {
```

Comment:
We should not export this. This type is only used in this file.

Reply:
Left this in by mistake, thanks, it's removed